notebook/app.py

```diff
@@ -169,15 +169,15 @@ def get(self, path=None):
 aliases = dict(base_aliases)
 
 
-class RetroApp(NBClassicConfigShimMixin, LabServerApp):
-    name = "retro"
-    app_name = "RetroLab"
-    description = "RetroLab - A JupyterLab Distribution with a retro look and feel"
+class JupyterNotebookApp(NotebookConfigShimMixin, LabServerApp):
```

Comment:
This class needs to be named something else than `NotebookApp`.

Otherwise the traits translation provided by `notebook_shim` would try to convert for example `NotebookApp.collaborative` to `ServerApp.collaborative`, but `collaborative does not exist on `ServerApp` but on `LabApp`.

```
$ jupyter notebook --no-browser
[C 2022-02-25 10:12:36.754 LabApp] Bad config encountered during initialization: Trait, expose_app_in_browser, not found.

```

cc @Zsailer to double check this is fine, which I believe it should, since `JupyterNotebookApp` inherits from `ServerApp`.